### PR TITLE
Annotations and Analysis Review for the Nullness Checker

### PR DESCRIPTION
**Pull request by Alicewillbe**
Please check the following 15 false positives.

### CODE REVIEW

**Comment by Alicewillbe** [comment]
Changes made for the Nullness_Lite option are finished, please review the Nullness Checker. @979216944

File: src/main/java/org/junit/experimental/max/MaxHistory.java [hunk 1]
```diff
@@ -81,7 +82,9 @@ private void save() throws IOException {
         stream.close();
     }
 
-    Long getFailureTimestamp(Description key) {
+    // Nullable Long returned because fFailureTimestamps.put()
+    // is called when a Failure occurs
+    @Nullable Long getFailureTimestamp(Description key) {
         return fFailureTimestamps.get(key.toString());
     }
```
Comment (AnnyKong) [inline src/main/java/org/junit/experimental/max/MaxHistory.java]: Do you mean by `fFailureTimestamps.get()`?

File: src/main/java/org/junit/experimental/max/MaxHistory.java [hunk 2]
```diff
@@ -94,6 +97,11 @@ boolean isNewTest(Description key) {
...
 
     Long getTestDuration(Description key) {
+        // [return.type.incompatible] FALSE_POSITIVE
+        // this method is accessible with in the package only,
+        // thus, not exposed to the users;
+        // the internal calls ensures key exist in fDurations
+        // and every test's duration is put in fDurations
         return fDurations.get(key.toString());
     }
```
Comment (AnnyKong) [inline src/main/java/org/junit/experimental/max/MaxHistory.java]: I would prefer more explanations for 
>the internal calls ensures key exist in fDurations

so, what are those internal calls?

File: src/main/java/org/junit/runners/ParentRunner.java [hunk 3]
```diff
@@ -500,6 +500,9 @@ private void validate(@UnknownInitialization ParentRunner<T> this) throws Initia
                 childrenLock.unlock();
             }
         }
+        // [return.type.incompatible] FALSE_POSITIVE
+        // filteredChildren cannot be uninitialized at this point,
+        // due to the code above
         return filteredChildren;
     }
```
Comment (AnnyKong) [inline src/main/java/org/junit/runners/ParentRunner.java]: I guess you mean `filteredChildren` cannot be null.

**Review by AnnyKong — CHANGES_REQUESTED** [review]
Some minor changes required.

**Review by AnnyKong — APPROVED** [review]

**Pull request merged**